app/services/Aws.scala

```diff
@@ -7,8 +7,11 @@ import software.amazon.awssdk.regions.Region
 object Aws {
   val credentialsProvider = AwsCredentialsProviderChain.builder().credentialsProviders(
     ProfileCredentialsProvider.builder.profileName("membership").build,
-    InstanceProfileCredentialsProvider.builder.asyncCredentialUpdateEnabled(false).build
-  )
+    InstanceProfileCredentialsProvider
+      .builder
+      .asyncCredentialUpdateEnabled(true)
+      .build
+  ).build
```

Comment:
Note that we now `build` the `AwsCredentialsProviderChain` once here. Otherwise we get errors from the underlying `ThreadPoolExecutor`